tests/autofill/autofill_component.test.ts

```diff
@@ -250,6 +251,16 @@ describe("Autofill component", () => {
     await nextTick();
     expect(firstViewport).toMatchObject(parent.model.getters.getActiveMainViewport());
   });
+
+  test("Autofill is not loaded when the grid selection does not have the focus", async () => {
```

Comment:
the test passes without the fix :/
which is normal since the previous code was checking the edition mode.
The needed test is for selection input

Reply:
added a test for selectionInput :)